app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/itemsList/itemsList.jsx

```diff
@@ -22,7 +22,15 @@ import { CURRENT_EXECUTION_ONLY } from 'pages/inside/stepPage/modals/editDefectM
 import { ItemsListHeader } from './itemsListHeader';
 import { ItemsListBody } from './itemsListBody';
 
-export const ItemsList = ({ testItems, selectedItems, setModalState, loading, optionValue }) => {
+export const ItemsList = ({
```

Comment:
I guess ItemList doesn't have the right section on its own? If so, rename rightSectionIsLess to smth like `isTinyView` or `isNarrowView` or etc.